include/effects/matrix/PatternWeather.h

```diff
@@ -174,6 +211,12 @@ class PatternWeather : public LEDStripEffect
             return KelvinToFarenheit(K);
     }
 
+    /**
+     * @brief Update the latitude and longitude for the
+     * selected city or zip code from the device configuration
+     * 
+     * @return bool - updated the location 

```

Comment:
Maybe replace with: "true if the lat/long location coordinates were updated"

Reply:
Agreed